src/_stories/execute/utilities.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from _stories.exceptions import StoryDefinitionError
+from _stories.returned import Failure
+from _stories.returned import Result
+from _stories.returned import Skip
+from _stories.returned import Success
+
+
+def validate_result_type(method, result_type, result):
+    if hasattr(method, "story_name"):
```

Comment:
This `if` statement is unnecessary since `BeginningOfStory` and `EndOfStory` will not be passed here.